pkgs/by-name/sc/scopehal-apps/package.nix

```diff
@@ -78,10 +84,31 @@ stdenv.mkDerivation {
     moltenvk
   ];
 
-  cmakeFlags = lib.optionals stdenv.hostPlatform.isDarwin [
-    "-DCMAKE_INSTALL_RPATH=${lib.strings.makeLibraryPath [ vulkan-loader ]}"
+  cmakeFlags = [
+    "-DNGSCOPECLIENT_VERSION=${version}"
+  ];
+
+  patches = [
+    ./remove-git-derived-version.patch
+  ]
+  ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    ./remove-brew-molten-vk-lookup.patch
```

Comment:
The darwin build will fail when trying to execute `brew` if this patch is not applied